docroot/profiles/custom/cgov_site/themes/custom/ncids_trans/templates/content/home_landing/paragraph--ncids-3-feature-card-row.html.twig

```diff
@@ -1,20 +1,13 @@
 {% if (content.field_row_cards['#items']|length > 0) %}
-  {% if content.field_container_heading[0] %}
-    <section class="usa-section usa-section--light cgdp-feature-card-row">
-      <div class="grid-container">     
-      {{ content.field_container_heading }}
-        <ul class="nci-card-group">
-          {{ content.field_row_cards }}
-        </ul>
-      </div>
-    </section>
-  {% else %}
-    <div class="usa-section usa-section--light cgdp-feature-card-row">
-      <div class="grid-container">
-        <ul class="nci-card-group">
-          {{ content.field_row_cards }}
-        </ul>
-      </div>
+  {% set id = paragraph.id() %}
+  {% set entity_type = paragraph.entityTypeID %}
+  {% set heading_id = "#{entity_type}-#{id}-heading" %}
+  <section class="usa-section cgdp-feature-card-row" aria-labelledby="{{heading_id}}">
+    <div class="grid-container">
+      <h2 id="{{heading_id}}" class="cgdp-feature-card-row__heading">{{ content.field_container_heading }}</h2>
```

Comment:
I messaged on Slack as well but whichever one you see first works. `content.field_container_heading` would probably need to change to `paragraph.field_container_heading.value` as the field twig template was deleted and nothing is currently being rendered out for the feature card row heading. I tested `paragraph.field_container_heading.value` locally and it works.